cypress.config.js

```diff
@@ -1,21 +1,21 @@
-const { defineConfig } = require("cypress");
-const faker = require('faker');
+const { defineConfig } = require('cypress');
+// const faker = require('faker');
 
 module.exports = defineConfig({
   e2e: {
     baseUrl: 'https://conduit.mate.academy/',
     setupNodeEvents(on, config) {
-      on('task', {
-        generateUser() {
-          const email = faker.internet.email();
-          const randomNumber = Math.floor(Math.random(1000) * 1000);
-          return {
-            username: faker.name.firstName() + randomNumber,
-            email: email.toLowerCase(),
-            password: '12345Qwert!'
-          };
-        }
-      });
+      // on('task', {
```

Comment:
You'll need that function. You'll also need a `generateArticle()` function which you should add below this one.